embedded-io-adapters/Cargo.toml

```diff
@@ -13,11 +13,13 @@ categories = [
 ]
 
 [features]
+default = ["digest"]
```

Comment:
should be fine to enable by default since this crate is also `no_std`

Reply:
i'd prefer to make it optional to not bloat up people's dependency trees and also because the 0.10 vs 0.11 thing